tubular/wordpress_api.py

```diff
@@ -0,0 +1,27 @@
+import requests
+from requests.auth import HTTPBasicAuth
+
+
+class WordPressAPI():
+    """
+    WordPress API to retire user data.
+    """
+
+    def __init__(self, wordpress_base_url, wordpress_username, wordpress_app_password):
+        self.base_url = wordpress_base_url
+        self.username = wordpress_username
+        self.password = wordpress_app_password
+
+    def retire_user(self, learner):
+        """
+        Retires the user with the given username in the WordPress.
+        """
+
+        url = "{}/wp-json/edly/v1/retirement".format(self.base_url)
```

Comment:
Can we have a constants/URLs file for adding API endpoints configuration instead of hardcoding directly into functions?

Reply:
no

Reply:
can you share the reason for that? and can we have this value: `/wp-json/edly/v1/retirement` in configuration/settings/contants?

Reply:
@asadmanzoor93  done.